packages/server/graphql/private/mutations/upgradeToTeamTier.ts

```diff
@@ -44,21 +42,20 @@ const upgradeToTeamTier: MutationResolvers['upgradeToTeamTier'] = async (
   const subOptions = {mutatorId, operationId}
 
   // AUTH
-  const viewerId = getUserId(authToken)
```

Comment:
The mutation is called from the server so the `viewerId` would always be `'aGhostUser'`